homeassistant/components/insteon/insteon_entity.py

```diff
@@ -0,0 +1,127 @@
+"""Insteon base entity."""
+import logging
+
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import (
+    DOMAIN,
+    INSTEON_ENTITIES,
+    SIGNAL_LOAD_ALDB,
+    SIGNAL_PRINT_ALDB,
+    STATE_NAME_LABEL_MAP,
+)
+from .utils import print_aldb_to_log
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class InsteonEntity(Entity):
+    """INSTEON abstract base entity."""
+
+    def __init__(self, device, state_key):
+        """Initialize the INSTEON binary sensor."""
+        self._insteon_device_state = device.states[state_key]
+        self._insteon_device = device
+        self._insteon_device.aldb.add_loaded_callback(self._aldb_loaded)
```

Comment:
Does this have side effects or does it just connect the callback?

Reply:
It just connects the callback.